lib/matplotlib/backends/backend_wx.py

```diff
@@ -901,13 +901,9 @@ def __init__(self, num, fig, *, canvas_class=None):
 
         self.figmgr = FigureManagerWx(self.canvas, num, self)
 
-        self.toolbar = self._get_toolbar()
-        if self.figmgr.toolmanager:
-            backend_tools.add_tools_to_manager(self.figmgr.toolmanager)
-            if self.toolbar:
-                backend_tools.add_tools_to_container(self.toolbar)
-        if self.toolbar is not None:
-            self.SetToolBar(self.toolbar)
+        toolbar = self.canvas.manager.toolbar
```

Comment:
Is that not?
```suggestion
        toolbar = self.figmgr.toolbar
```

Reply:
Yes, but I have some later plans to get rid of the (non-standard) figmgr as well...